spec/menu_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dish_klass) {double(:dish_klass)}
+  let(:dish) {double(:dish)}
+  let(:dish2) {double(:dish)}
+  let(:dish_list) { [ {name: :name_arg, price: :price_arg},
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.